src/EventListener/ShippingExportEventListener.php

```diff
@@ -135,15 +130,14 @@ public function autoExport(PaymentInterface $payment): void
             return;
         }
 
-        $event = new ExportShipmentEvent(
-            $exportObject,
-            $this->flashBag,
-            $this->entityManager,
-            $this->filesystem,
-            $this->translator,
-            $this->shippingLabelsPath
-        );
+        $exportShipmentEvent = new ResourceControllerEvent($exportObject);
```

Comment:
Why this change?

Reply:
![1](https://github.com/user-attachments/assets/a6fa791b-4052-48de-82ca-1eb70a4f6a59)
![2](https://github.com/user-attachments/assets/d9a16f80-c62b-4139-b656-20576e1e607a)

`The ExportShipmentEvent is deprecated since Sylius 1.8 and will be removed. Use \Sylius\Bundle\ResourceBundle\Event\ResourceControllerEvent instead`